OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3538,91 +3545,244 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            </handNote>
+                            
+                            <handNote xml:id="h2" script="Ethiopic" corresp="#a1 #a2">
+                                <date notAfter="1868"/>
+                                <seg type="ink">Black.</seg>
                             </handNote>
                         </handDesc>
                         
                        <!-- <decoDesc>
-                         
+                           captions:
+                           3v: ሥሉስ፡ ቅዱስ፡
+                         4r: ካህናተ፡ ሰማይ፡
+                         5v: ቅዱስ፡ ጊዮርጊስ
+                         6r: ቅዱስ፡ ሚካኤል፡ ምስለ፡ ፍቁር፡ ወልዳ፡ ቅዱስ፡ ገብርኤል፡
+                         ከመ፡ ተማኅፀነ፡ <persName ref="PRS12911KidanaK"><roleName type="title">አባ፡</roleName> ኪዳነ፡ ክርስቶስ፡</per>
+                         19v: ኪዳነ፡ ምሕረት፡
+                         20r: ዘከመ፡ አዕረፈት፡ እግዝእትነ፡ ማርያም፡
+                         20v: ፍልሰታ፡
+                         29v: ዘከመ፡ አጥበወቶ፡ እግዝእትነ፡ ለእግዚእነ፡
+                         82v: አቡነ፡ አቢብ፡
+                         ዘከመ፡ ተማኅፀነ፡ <persName ref="PRS13031WaldaA">ወልደ፡ አቢብ፡</persName>
+                         83r: አቡነ፡ ተክለ፡ ሃይማኖት።
+                         117v: ገነት፡
+                         ዓቃቤ፡ ገነት፡ ፈያታይ፡ ዘየማን፡
+                         118r: ዘከመ፡ ቀሠፍዎ፡ ለእግዚእነ፡
+                         118v: ኵርዓተ፡ ርእሱ፡
+                         ዘከመ፡ ተማኅፀነ፡ ጎርጎርዮስ፡
+                         119r: ዘከመ፡ ተማኅፀነ፡ <persName ref="PRS12911KidanaK"><roleName type="title">አባ፡</roleName> ኪዳነ፡ ክርስቶስ፡</per>
+                         119v: ዘከመ፡ ወደይዎ፡ ውስተ፡ መቃብር፡
+                         120r: ትንሣኤ፡ አዳም፡ ሔዋን፡
+                         120v: ዕርገት፡
+                         121r: ጰራቅሊጦስ፡
+                         121v: ምጽአት፡
                         </decoDesc>-->
                         
                         <additions>
                             <list>
-                                <item xml:id="a1">
+                                <item xml:id="a1" corresp="#h2">
                                     <locus target="#5r"/>
-                                    <desc type="Inventory">List of liturgical vessels and books belonging to a church of  Madḫāne ʿĀlam.</desc>
-                                    <q xml:lang="am">ኍልቈ፡ ንዋዮ፡ ቅድሳት፡ ዘመድኃኔ፡ ዓለም፡
+                                    <desc type="Inventory">Amharic list of objects, mainly liturgical vessels and books belonging to a 
+                                        church identified as
+                                        Madḫāne ʿĀlam, given by <persName ref="PRS12911KidanaK"/>,
+                                        ending with a condemnation. Among the objects is also a drawing on parchment.</desc>
+                                    <q xml:lang="am"><foreign xml:lang="gez">ኍልቈ፡ ንዋዮ፡ ቅድሳት፡ ዘመድኃኔ፡ ዓለም፡</foreign>
                                         <persName ref="PRS12911KidanaK"><roleName type="title">አባ፡</roleName> ኪዳነ፡ ክርስቶስ፡</persName> የሰጡ፡
```

Comment:
Sorry, it is not yet clear to me: As there is written that the objects were "given" by Kidane Krestos it is not only a "list of objects" but also a statement about a donation, isn't it?

Reply:
Yes, but since it starts with ኍልቈ፡ ንዋየ the inventory character is more prominent. Do you have a suggestion for a formulation?

Reply:
maybe: "list of donated objects" 
I would clearly state that they were donated also in the description, did you?